pkg/manager/manager.go

```diff
@@ -321,9 +324,9 @@ func (m MediaManager) AddIndexer(ctx context.Context, request AddIndexerRequest)
 	return indexer, nil
 }
 
-// DeleteIndexerRequest describes what is required to delete an indexer
+// DeleteIndexerRequest request to delete an indexer
 type DeleteIndexerRequest struct {
-	ID *int `json:"id" yaml:"id" mapstructure:"id"`
+	ID *int `json:"id" yaml:"id"`
```

Comment:
How come this is a pointer? It's required right

Reply:
I think I did that so I could check if it was nil vs zero value of 0